core/dbt/parser/manifest.py

```diff
@@ -614,6 +629,9 @@ def track_project_load(self):
             "is_partial_parse_enabled": (
                 self._perf_info.is_partial_parse_enabled
             ),
+            "is_static_analysis_enabled": flags.USE_EXPERIMENTAL_PARSER,
```

Comment:
Could we add `is_static_analysis_enabled` to the output of `dbt parse` (`target/perf_info.json`) as well?